extensions/Needinfo/template/en/default/hook/request/email-after_summary.txt.tmpl

```diff
@@ -8,6 +8,27 @@
 
 [% RETURN UNLESS flag && flag.type.name == 'needinfo' && flag.status == '?' %]
 ---
---- This request has set a needinfo flag on the [% terms.bug %].
---- You can clear it by logging in and replying in a comment.
+--- Hello from [% terms.Bugzilla %], the [% terms.bug %] tracker for Mozilla and Firefox:
+---
+--- Someone has asked for you for information about this [% terms.bug %].
+---
+--- Please go to [% urlbase %]show_bug.cgi?id=[% bug.bug_id %] to respond
+--- to the question below.
+[% IF flag.requestee.login_name == bug.reporter.login_name %]
+---
+--- Since you reported this [% terms.bug %], then the person asking the
+--- question needs more information from you to understand it.
+[% END %]
+---
+--- Responding to the question will enable developers to take further
+--- action on this [% terms.bug %].
+---
+--- Please log into [% terms.Bugzilla %] and respond as soon as possible
+--- so developers may take action on this [% terms.bug %].
+---
+--- If you have questions about responding to needinfo requests, please
+--- see https://mozilla.tld/some/path/to/documentation.
```

Comment:
I'm not convinced this is needed, given the target audience of this email are people who already know how to use Bugzilla.

In any event, this can't land as-is until a valid URL is provided.

Reply:
Per discussion my Monday morning, I wanted to clarify what to do if you needed to pass the needinfo to someone else, etc, so I linked to the User Guide on wiki.mo and updated the section on needinfo from _this is good and you should use it_.